EIPS/eip-remove_blake2f_precompile.md

```diff
@@ -0,0 +1,47 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7266
```
Assigning EIP number using the PR number.  Please also update the filename

Reply:
resolved via https://github.com/ethereum/EIPs/pull/7266/commits/40f46ad6637e0d7e5f86bd8b51373f50e1ad57db